Backend/Models/Database/Pharmacy.cs

```diff
@@ -21,5 +21,8 @@ public class Pharmacy
 
         [Required]
         public ICollection<Register> Registers { get; set; }
+
+        [Required]
```

Comment:
According to our class diagram, this relation is optional, so we do not need `[Required]` field.

Reply:
Fixed